static/build/App/Clock/config.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
This whole directory `static/build/App/Clock` should be moved to `static/packages/App/Set emulated time/App/Clock`

Reply:
done